HeterogeneousCore/AlpakaInterface/interface/blas/alpakaBlasCore.h

```diff
@@ -0,0 +1,277 @@
+#ifndef HeterogeneousCore_AlpakaInterface_interface_blas alpakaBlasCore_h
+#define HeterogeneousCore_AlpakaInterface_interface_blas_alpakaBlasCore_h
+
+#include "HeterogeneousCore/AlpakaInterface/interface/config.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/VecArray.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/blas/alpakaReducer.h"
+
+namespace cms::alpakatools{
+  namespace blas{
```

Comment:
I suggest to keep it as is for now, usually I consider chaining less convenient since one may want to clearly distinguish between different namespaces : say, I may define something outside blas namespace in the future. (Of course one could just create another namespace scope, but for me it'd look less cleaner, though it could be a matter of taste)